tests/timeseries/test_resampling.py

```diff
@@ -173,6 +173,88 @@ async def test_helper_buffer_too_big(
     assert helper._buffer.maxlen == DEFAULT_BUFFER_LEN_MAX
 
 
+async def test_resampling_window_size_is_constant(
+    fake_time: time_machine.Coordinates, source_chan: Broadcast[Sample]
+) -> None:
+    """Test resampling window size is consistent."""
+    timestamp = datetime.now(timezone.utc)
+
+    resampling_period_s = 2
+    expected_resampled_value = 42.0
+
+    resampling_fun_mock = MagicMock(
+        spec=ResamplingFunction, return_value=expected_resampled_value
+    )
+    config = ResamplerConfig(
+        resampling_period_s=resampling_period_s,
+        max_data_age_in_periods=1.0,
+        resampling_function=resampling_fun_mock,
+        initial_buffer_len=4,
+    )
+    resampler = Resampler(config)
+
+    source_recvr = source_chan.new_receiver()
+    source_sendr = source_chan.new_sender()
```

Comment:
No need to change now, but it's a bit weird that the last `e` in both those variables is worth leaving out, might as well type the full name? :D